extensions/kube-context/src/extension.ts

```diff
@@ -48,45 +48,42 @@ export async function updateContext(
   // read the kubeconfig file using fs
   const kubeConfigRawContent = await fs.promises.readFile(kubeconfigFile, 'utf-8');
 
-  if (kubeConfigRawContent.length === 0) {
-    console.error(`Kubeconfig file at '${kubeconfigFile}' is empty. Skipping.`);
-    return;
-  }
-
   // parse the content using jsYaml
   const kubeConfig = jsYaml.load(kubeConfigRawContent);
 
   // get the current context
-  const currentContext = kubeConfig['current-context'];
+  const currentContext = kubeConfig ? kubeConfig['current-context'] : undefined;
 
   // get all contexts
-  const contexts: KubeContext[] = kubeConfig['contexts'];
+  const contexts: KubeContext[] = kubeConfig?.['contexts'] ? kubeConfig['contexts'] : [];
+
+  if (contexts.length > 0) {
```

Comment:
do we need that check as iterating on an empty array won't do anything

Reply:
The clue here is that if there are no context then no tray item menu will be registered (as previously when kubeconfig file was empty) thus the test

Reply:
so probably do 

```ts
console.log(`Kubeconfig file at '${kubeconfigFile}' has no contexts. Skipping.`);
if (contexts.length === 0) {
return;
}
```